### PR TITLE
Make edit source command always use boolean constants

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: qpc/source/edit.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def __init__(self, subparsers):
                                  help=_(messages.SOURCE_PORT_HELP))
         self.parser.add_argument('--ssl-cert-verify',
                                  dest='ssl_cert_verify',
-                                 choices=['True', 'False'],
+                                 choices=source.BOOLEAN_CHOICES,
```
Comment (chambridge) [inline qpc/source/edit.py]: Should we maybe include test cases this time? 🤷‍♂️

**Comment by kholdaway** [comment]
:trollface:

**Comment by chambridge** [comment]
```
flake8 . --ignore D203 --exclude quipucords/api/migrations,docs,build,.vscode,client,venv,deploy
./qpc/source/tests_source_add.py:274:80: E501 line too long (81 > 79 characters)
./qpc/source/tests_source_add.py:283:33: E128 continuation line under-indented for visual indent
./qpc/source/tests_source_add.py:284:33: E128 continuation line under-indented for visual indent
./qpc/source/tests_source_add.py:288:37: E128 continuation line under-indented for visual indent
```

**Comment by kholdaway** [comment]
Fixed

**Pull request merged**